Exercise_A/index.html

```diff
@@ -6,33 +6,217 @@
+
+      <div class="d-flex justify-content-center align-content-center whiteText" id="headerText">
+        <h1 id="headerTitle">HEY LOOK, IT'S EVERY BOOTSTRAP WEBSITE EVER</h1>
+      </div>
+      <hr id="orangeHR">
+      <div class="d-flex justify-content-center align-content-center whiteText">
+        <p>Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring all the same things as those pages!</p>
+      </div>
+
+
+      <div class="d-flex justify-content-center align-content-center">
+        <a href="#about" class="btn btn-lg" style="color:white; background-color: #f05f40" href="#" role="button" id="learnMore">This button makes the page scroll</a>
+      </div>
+   </div>
+   <!-- First orange div section-->
+   <div class="container-fluid firstOrangeDiv">
+     <div class="d-flex justify-content-center align-content-center whiteText">
+       <h3 id="about">Want to make an original website for yourself?</h3>
+     </div>
+     <hr id="whiteHR">
+     <div class="d-flex justify-content-center align-content-center whiteText" >
+        <p>Forget that! Who would ever want to put in all of that effort for a website? Just open up your web browser
+          and type "bootstrap template" into your favorite search engine, like Yahoo! or Bing, and you're on your way!
+          There are hundreds of templates to choose from, but go ahead and pick this same exact template from the first result on google,
+          edit a few lines, and you're on your way! No one will notice!</p>
+     </div>
+     <div class="d-flex justify-content-center align-content-center">
+      <a href="http://lmgtfy.com/?q=bootstrap+template" target="_blank" class="btn btn-lg" style="background-color: white" href="#" role="button" id="learnMore">GOOGLE THAT SHIT</a>
+    </div>
+    </div>
+
+
+
+    <!--Four icons div-->
+    <div class="container-fluid" id="fourIconsDiv">
+
+          <div id="fourIcons" class="d-flex justify-content-center align-content-center">
+            <h3>So Fucking Creative</h3>
+          </div>
+      </div>
+      <hr id="orangeHR">
+      <div class="row">
+      
+            <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="iconFlex">
+              <div class="secondBox">
+              <i class="fas fa-globe-americas fa-3x" style="color: #f05f40"></i>
+              <h3>This Fucking Template</h3>
+              <p>Guaranteed to use the same fucking template that every other bootstrap website uses, downloaded straight from The Web™</p>
+            </div>
+            </div>
+      
+            <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="iconFlex">
+              <div class="secondBox">
+              <i class="fas fa-paper-plane fa-3x" style="color: #f05f40"></i>
+              <h3>This Set of Four Icons</h3>
+              <p>Look at this cool set of four icons describing different things about us! We use four, because it's the default.</p>
+            </div>

```

Comment:
Nice job with the column system here
Try to stick to one colour convention throughout you html, css etc.